docker/mainnet/docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
   sbtc-signer:
     container_name: sbtc-signer
     restart: on-failure
-    image: blockstack/sbtc:signer-0.0.9-rc6@sha256:44d2c828c6452f75ea2fabe5160ba95a9f944c0fcaeeaea5523dff81da8bc342
```

Comment:
Can you add a comment also here that this could be outdated and might need to be bumped?

Reply:
Thank you, this can be resolved (I can't do it).